src/hooks/useSetFocus/__tests__/useSetFocus.jest.js

```diff
@@ -0,0 +1,89 @@
+import { act, cleanup, renderHook } from "@testing-library/react-hooks";
+import useSetFocus from "../index";
+
+describe("useSetFocus", () => {
+  let element;
+  const focusCallback = jest.fn();
+  const blurCallback = jest.fn();
+
+  describe("click", () => {
+    afterEach(() => {
+      element.remove();
+      cleanup();
+    });
+
+    it("default isFocused value is false", () => {
+      const { result } = renderHookForTest();
+      expect(result.current.isFocused).toBe(false);
+    });
+
+    it("should call focusCallback when focusing the element", () => {
```

Comment:
this test  check 2 different things (not only focusCallback) let's separate it to 2 tests